### ISSUE TITLE
Styles of buttons are inconsistent

### ISSUE THREAD

**Issue by shuuji3** [kubernetes/website#22892]
**This is a Bug Report**

**Problem:**
The styles of buttons are inconsistent. Proper style is applied only to buttons on a few pages.

A. Kubernetes style
![a-kubernetes-style](https://user-images.githubusercontent.com/1425259/89099230-bc3b3a80-d428-11ea-965d-601b7f5c1850.png)
Example: [Using Minikube to Create a Cluster | Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/)

B. Docsy default style (should be changed)
![b-docsy-default](https://user-images.githubusercontent.com/1425259/89099228-bba2a400-d428-11ea-8010-c48d75420369.png)
Example: [Production environment | Kubernetes](https://kubernetes.io/docs/setup/production-environment/)

**Proposed Solution:**
Apply the first style (A. Kubernetes style) to all the buttons.

**Page to Update:**
We need to fix some stylesheet problem.

**Comments:**
In the case of pages of the A style, it seems that two stylesheets are applied:
- https://kubernetes.io/docs/tutorials/kubernetes-basics/public/css/styles.css
- src/themes/docsy/assets/vendor/bootstrap/scss/_buttons.scss

`styles.css` is overriding the default docsy style of `_buttons.scss`.

However, for pages of the B style, only the default docsy style (`_buttons.scss`) is applied.

/area web-development

**Comment by blueabysm** [comment]
/assign
/good-first-issue

Let me handle this if you don't mind :) @shuuji3

**Comment by shuuji3** [comment]
@blueabysm No worries at all. Thank you! 🙂

**Comment by blueabysm** [comment]
@shuuji3 
I found only routes under `/<lang>/docs/tutorials/kubernetes-basics/` would apply style `/docs/tutorials/kubernetes-basics/public/css/styles.css`, which is the file `/path/to/website/content/en/docs/tutorials/kubernetes-basics/public/css/styles.css` shared among pages in other languages. Different from other pages written in markdown and parsed by Docsy, these pages are all HTML files, manually referring `styles.css` file in a `link` tag, [for example](https://github.com/kubernetes/website/blob/5f87a282b1803a4062df11e97dbb71b9ebec6210/content/fr/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html#L12).
```

I have no idea the exact purpose the former contributors doing this. Related commit messages tells not much. Maybe I can pick up the button styles and put them into scss files used by docsy. I'm not quite sure that it's a good option. Let me have a test.

**Comment by shuuji3** [comment]
@blueabysm Thank you for your investigation. In fact, I don't know the historical reason for this well... Looking at the history of [https://github.com/kubernetes/website/commits/master/content/en/docs/tutorials/kubernetes-basics/public/css/styles.css](r/content/en/docs/tutorials/kubernetes-basics/public/css/styles.css), I found it came from around 2018, long before Docsy introduced.

I also noticed the third type of style on https://kubernetes.io/docs/home/. This button style came from https://github.com/kubernetes/website/tree/master/assets/scss/_base.scss. These styles are introduced 2 months ago for the latest website renewal, so maybe we should use this style for all the buttons.

**Comment by blueabysm** [comment]
@shuuji3 Great! But buttons on page `/docs/home` have different CSS classes. Maybe it's what they are designed to be.🙂
**Seems we need to know which style is supposed to be in use first**, and then we can talk about how to achieve this. I personally suggest to hold on this issue until someone could confirm, but I don't know who is the proper person I should @ 😂. Maybe the leads of this repo?

Would you please help us with this issue?  I'm not sure which style I should use as the unified style. Thanks a lot! @zacharysarah

**Comment by shuuji3** [comment]
I found another inconsistent style on the page [Set up Ingress on Minikube with the NGINX Ingress Controller](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/).

**Current style**
![v1 17](https://user-images.githubusercontent.com/1425259/90601898-d7cd8000-e233-11ea-9c6e-69491b136ecc.png)

**Previous style (v1.17)**

![master](https://user-images.githubusercontent.com/1425259/90601901-d8fead00-e233-11ea-9a9f-461fb0ae98e5.png)

**Comment by shuuji3** [comment]
I've created the PR to fix the above issue.

**Comment by shuuji3** [comment]
FYI: "Create an Issue" and "Edit This Page" buttons had been moved to the right sidebar so we don't need to change the style for them.

**Comment by sftim** [comment]
Fixed in PR #24899 
/close
